packages/2018-transportation-systems/src/state/import-adapter.js

```diff
@@ -0,0 +1,11 @@
+// A redux action utility for making a thunk based on an async file import
+// ex. const someJsonInRepo = importAdapter('../assets/some.json', { start, success });
+//     dispatch(someJsonInRepo())
+const importAdapter = (importPromise, { start, success }) => () => (dispatch) => {
+  dispatch(start());
+  return importPromise.then((res) => {
```

Comment:
Continuing the thought above about making this an `apiAdapter`, this line would become something like:

```js
return axios.get(url).then((res) => {
  dispatch(success(res));
}).catch((err) => {
  dispatch(failure(err));
});
```